azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -14,10 +14,10 @@
     SAMPLING_RATIO_ARG,
 )
 from azure.monitor.opentelemetry._types import ConfigurationValue
-from azure.monitor.opentelemetry._vendor.v0_38b0.opentelemetry.instrumentation.dependencies import (
+from azure.monitor.opentelemetry._vendor.v0_39b0.opentelemetry.instrumentation.dependencies import (
```

Comment:
Did we release a version of the distro which vendored v0.38b0 before?

Reply:
No. Can do.

Reply:
https://github.com/microsoft/ApplicationInsights-Python/pull/296

Reply:
That's fine that we didn't. I'm just wondering if, before we go GA, should we have one folder that we keep updating, or should we add a new vendored folder for each version of the instrumentations? i.e. have a vendor folder for 0.38b0 and one for 0.39b0 and so on for every release

Reply:
I think one folder is ultimately better. Far less unused code. Easier to notice iterations. But if there's ever a breaking bug where we need to keep more than one, we can.